### DISCUSSION TITLE
Want to get the star catalog file used by Stellarium

### DISCUSSION THREAD

**Discussion by tsuiwade** [Stellarium/stellarium#2334] [Q&A] [answered]
I know Stellarium’s star catalogues are based on Hipparcos ,Tycho 2 and NOMAD catalogues.

I want to get the star catalog file that Stellarium is using, I want to get the location information ( RA / Dec ) of the stars used by stellarium, where can I find it? 

Thank you!

**Comment by alex-w** [comment, marked as answer]
Please see Stellarium User Guide and source code of planetarium

**Comment by kenyasue** [comment]
Hi

I'm trying to achieve the same stuff. I read the PDF, I can get x0 and x1. But I stacked to convert the x0 and x1 to RA/DEC. 
I'm trying to get the logic from the source code, but still no luck... Can someone let me know where I can find the logic ?

**Reply by gzotti** [reply]
ZoneArray.{hc}pp

**Reply by kenyasue** [reply]
Thanks !

**Reply by kenyasue** [reply]
I realize that you have this approach to save machine power and increase usability. Changing it to RA/DEC doesn't make sense. I'm trying to make my own planetarium software for VR headset.
